README.rst

```diff
@@ -61,6 +61,20 @@ contributions!
 
 See the `developer docs`_ and the `contribution docs`_ for more information.
 
+
+License
```

Comment:
this section you have created is going to exist in the `NOTICE` file, instead of the README

here is an example NOTICE file:

https://www.apache.org/licenses/example-NOTICE.txt

in so much as we are able, and in so much as it makes sense for us as a non ASF product, we are following the ASF guidelines on project licensing. you can read more about those here:

https://www.apache.org/dev/apply-license.html

Reply:
@eirinikos can confirm that putting the info in a NOTICE file will not confuse Licensee.